### PR TITLE
Speech Synth Demo

### PR DESCRIPTION
**Pull request by ststimac**
Stylized speech synthesizer demo. 

The purpose behind the parrot illustrations: they seem random at first, but to create an ‘experience’ in the demo, I tried to come up with
something that could relate to speech & having speech repeated. Parrots repeat what you say, so they’re our speech synthesizer mascots. Where text is entered is now a speech bubble, so we have a talking parrot.

Other additions: Created a logo, moved some things around slightly,
adjusted coloring, and added a sound wave graphic at the bottom of the
screen to anchor the page.

### CODE REVIEW

**Comment by ststimac** [comment]
@molant this is ready to merge. 

Whenever you have a moment to approve it. Jerry has his accompanying blog post ready to go. Thanks!

**Comment by molant** [comment]
@melanierichards have you taken a look at this?

**Comment by melanierichards** [comment]
Demos get embedded into our site, which adds CSS, so add `<link rel="stylesheet" type="text/css" href="https://edgeportal.blob.core.windows.net/media/demotemplate.css">` to the head of the demo and make sure that doesn't goof up any of the styles.

Another cascading (hehehehe) piece of feedback is that our CSS classes are `dash-separated` and JS IDs are `camelCase`. Some CSS selectors in this demo are camel cased, so those should be changed to dash-separated.

@molant and I want to experiment with prepending any classes/IDs used by scripts with `js`. So, for example `#jsMyCoolElement`. Maybe we could try that out w/ this demo?

**Comment by ststimac** [comment]
Adding `<link rel="stylesheet" type="text/css" href="https://edgeportal.blob.core.windows.net/media/demotemplate.css">`completely demolished the styling of the demo resulting in a general loss of layout/structure.

**Comment by melanierichards** [comment]
That is what I was afraid of :( Unfortunately better to fix it now than have it blow up when it's in the site.

**Comment by ststimac** [comment]
@molant @melanierichards the only thing I haven't done yet is the JS test you want to do. Just to clarify, I would only be editing the HTML file when I add 'js' to the classes/IDs used by scripts? Or would that also fall to the CSS file as well?

**Comment by molant** [comment]
@ststimac you will need to rename in HTML, CSS and JavaScript

**Comment by melanierichards** [comment]
I would think that any classes/IDs used as hooks for JavaScript should not be used for the CSS as well. So for example: `<div id="js-ThingAMaBob" class="semantic-style-class">`

File: speechsynthesis/styles/style.css [hunk 1]
```diff
@@ -1,119 +1,183 @@
...
-#supportMessage strong {
-  color: #006400;
+#jsSupportMessage strong {
+	color: #4c9d4c;
 }
 
-#supportMessage.unSupported strong {
-  color: #880000;
+#jsSupportMessage.unSupported strong {
+	color: #800;
```
Comment (ststimac) [inline speechsynthesis/styles/style.css]: @melanierichards @molant I didn't add a class here to the support message. I wanted to check first, would removing the ID here affect how the Javascript functions? The way I understand it and please correct me if I'm wrong, is that the JS on line 27-32 is using the ID and then adding the 'unsupported' class...so if I removed the ID '#jsSupportMessage' and replaced it with a class....wouldn't that interfere with the JS? (I am still learning JS so I may be way off here).

File: speechsynthesis/styles/style.css [hunk 2]
```diff
@@ -1,119 +1,183 @@
...
 
-#supportMessage strong {
-  color: #006400;
+#jsSupportMessage strong {
+	color: #4c9d4c;
 }
 
-#supportMessage.unSupported strong {
-  color: #880000;
+#jsSupportMessage.unSupported strong {
```
Comment (ststimac) [inline speechsynthesis/styles/style.css]: @melanierichards @molant I didn't add a class here to the support message. I wanted to check first, would removing the ID here affect how the Javascript functions? The way I understand it and please correct me if I'm wrong, is that the JS on line 27-32 is using the ID and then adding the 'unsupported' class...so if I removed the ID '#jsSupportMessage' and replaced it with a class....wouldn't that interfere with the JS? (I am still learning JS so I may be way off here).

**Comment by ststimac** [comment]
@molant @melanierichards Kyle has the blog post for this speech demo scheduled to go out on June 1 (Wed). Will need to be approved and merged before then.

File: speechsynthesis/styles/style.css (hunk 2 above)
Comment (molant) [inline speechsynthesis/styles/style.css]: @ststimac that will interfere. Also I believe the casing is not the right one. Should all be lowercase dash separated. Did you fixed all the stylelint issues?

Comment (ststimac) [inline speechsynthesis/styles/style.css]: @molant yes, all the stylelint issues are fixed. 

If the CSS ID is the same as the Javascript ID, that needs to be dashed and not camelCase?

**Comment by ststimac** [comment]
@molant changed all the IDs to js-lowercase

File: speechsynthesis/index.html [hunk 3]
```diff
@@ -1,45 +1,56 @@
 ﻿<!DOCTYPE html>
 <html>
 <head>
-    <meta charset="UTF-8">
-    <title>Speech Synthesis API</title>
-    <link rel="stylesheet" href="styles/style.css">
+	<meta charset="UTF-8">
+	<title>Speech Synthesis API</title>
+	<link rel="stylesheet" type="text/css" href="https://edgeportal.blob.core.windows.net/media/demotemplate.css">
+	<link rel="stylesheet" href="styles/style.css">
```
Comment (molant) [inline speechsynthesis/index.html:10]: We need to add a few tags so we get the right data published.

``` html
<title>@Supports - Test Whether Certain CSS Properties and Values are Supported</title>
<meta name="description" content="NEED A DESCRIPTION">
<meta name="keywords" content="misc, SOME, KEYWORDS, HERE">
<meta name="author" content="jdsmith3000, ststimac">
```

@jdsmith3000 can you help @ststimac fill up the missing fields above? Mainly description and keywords.

**Comment by ststimac** [comment]
@molant meta info has been added

**Comment by ststimac** [comment]
@molant volume and rate sliders aren't showing up in Edge. Give me a moment to see if I can figure out what's up.

**Pull request merged**